### PR TITLE
Paginate in monitor and frontend (Geneva actions)

### PR DESCRIPTION
**Pull request by jim-minter**
### What this PR does / why we need it:

Should fix BBM OOM kills seen due to too many ReplicaSet objects on a managed cluster.  Also a frontend OOM kill when trying to list the objects via Geneva action :-(

### Test plan for issue:

By inspection (view PR with whitespace diff off).

### Is there any documentation that needs to be updated for this PR?

No.

### CODE REVIEW

**Comment by jim-minter** [comment]
@mjudeikis @bennerv ptal

File: pkg/frontend/adminactions/kubeactions.go
Comment (jim-minter) [inline pkg/frontend/adminactions/kubeactions.go:95]: I don't love this, but I think it's pragmatic for now.

Comment (bennerv) [reply pkg/frontend/adminactions/kubeactions.go:95]: The JMESPath query runs after the data is fetched correct?  

I think we have some queries that check for pods in crashloopbackoff or without limits which this might inhibit until we get over to JIT auto approval on the portal.

File: pkg/frontend/adminactions/kubeactions.go
Comment (gvanderpotte) [inline pkg/frontend/adminactions/kubeactions.go:106]: Should not we display a warning saying only the first 1000 items (provided the order of return is deterministic or can be made deterministic) are returned instead of an error and no result displayed ?

File: pkg/util/clusterdata/ingress_profiles_task.go
Comment (gvanderpotte) [inline pkg/util/clusterdata/ingress_profiles_task.go:116]: Sounds unrelated with the rest of the PR, is it ?

File: pkg/util/clusterdata/worker_profiles_task.go
Comment (gvanderpotte) [inline pkg/util/clusterdata/worker_profiles_task.go:100]: Same here

File: pkg/frontend/adminactions/kubeactions.go
Comment (gvanderpotte) [reply pkg/frontend/adminactions/kubeactions.go:95]: Sorry for naive question, can those const values like the number of records per page be part of a conf file so that we can somehow override/tweak them and avoid a full delivery of the RP ?

**Review by gvanderpotte — COMMENTED** [review]
A few questions and one remark on Ingress/Worker profile sorting (which sounds off topic here), but apart from that LGTM

**Review by mjudeikis — APPROVED** [review]
I think we need to do more work on this in the next few sprints...  Maybe split monitoring to separate container?

**Pull request merged**
